doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -96,14 +98,27 @@ The following list summarizes the most important changes inherited from the upst
 * Added:
 
    * Support for the Intermittently Connected Devices (ICD) Management cluster.
-   * The Kconfig options :kconfig:option:`CONFIG_CHIP_ICD_IDLE_MODE_INTERVAL`, :kconfig:option:`CONFIG_CHIP_ICD_ACTIVE_MODE_INTERVAL` and :kconfig:option:`CONFIG_CHIP_ICD_CLIENTS_PER_FABRIC` to manage ICD configuration.
+   * The Kconfig options :kconfig:option:`CONFIG_CHIP_ICD_IDLE_MODE_DURATION`, :kconfig:option:`CONFIG_CHIP_ICD_ACTIVE_MODE_DURATION` and :kconfig:option:`CONFIG_CHIP_ICD_CLIENTS_PER_FABRIC` to manage ICD configuration.
+   * New device types:
+
+     * Refidgerator
+     * Room Air Conditioner
+     * Dishwasher
+     * Laundry Washer
+     * Robotic Vacuum Cleaner
+     * Smoke CO Alarm
+     * Air Quality Sensor
+     * Air Purifier
+     * Fan
```

Comment:
I believe these should be added to the [Matter Data Model and device types](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/protocols/matter/overview/data_model.html#matter-data-model-and-device-types) page.

Reply:
Yes, good catch, I forgot we have such page. I will update the list and push new change soon.